textinput.go

```diff
@@ -31,6 +31,10 @@ type textinputPlugin struct {
 	selectionExtent int
 }
 
+type keyboardShortcutsGLFW struct {
```

Comment:
I'm not sure why this is here, textinput.go already has this structure: https://github.com/go-flutter-desktop/go-flutter/blob/a6ff8ec870f4cae597059f4405692c3741c4e390/textinput.go#L31-L34